mix.exs

```diff
@@ -50,10 +50,9 @@ defmodule Elixometer.Mixfile do
 
   defp deps do
     [
-      #lager 3.2.1 is needed for erl19 because of
-      # https://github.com/basho/lager/pull/321
-      {:lager, ">= 3.2.1", override: true},
-      {:exometer_core, "~>1.4.0"},
+      # Force rebar so that setup can build, does not build with rebar3 base compiler
+      {:setup, "1.8.4", override: true, manager: :rebar},
```

Comment:
You don't need to override `setup` here. `exometer_core` now fetches 1.8.4 version and it should work by default.

Reply:
Can you provide versions you tried for this because it did not work for me without override for:
* Elixir 1.5.1
* Erlang/OTP 20.0
* Rebar3 3.3.6

Reply:
* Elixir 1.4.5
* Erlang/OTP 20.0
* Rebar3 3.3.6

I've prepared a simple application with the last exometer_core: https://github.com/surik/exo_setup_example

Reply:
The example project doesn't work in Elixir 1.5. @tsloughter do you know an approach that wouldn't require mix override? Or know what needs to be done to make this work. I am happy to look into rebar3 and/or elixir/mix to improve the interop.

Reply:
I'm curious what changed in Elixir/mix that it stopped working with rebar3?

Reply:
I've just figured out why it works for me even with Elixir 1.5.  I have `export PATH=$PATH:/Users/surik/.mix/` with `rebar` and `rebar3`.

Reply:
@tsloughter this is the error FWIW:
```
./rebar skip_deps=true escriptize                                                                                                                   
make: ./rebar: No such file or directory                                                                                     
make: *** [escriptize] Error 1                                                                
===> Hook for compile failed!                                                     
                                                                                    
** (Mix) Could not compile dependency :setup, "/.mix/rebar3 bare compile --paths "/code/elixometer/_build/test/lib/*/ebin"" 
command failed. You can recompile this dependency with "mix deps.compile setup", update it with "mix deps.update setup" or clean it with "mix deps.c
lean setup"
```